src/prisma/queries/init/guideLoop.ts

```diff
@@ -2,16 +2,16 @@ import { Prisma } from '@prisma/client';
 
 export const INITIAL_GUIDE_LOOP: Prisma.GuideLoopCreateInput = {
   m2TipTiltEnable: true,
-  m2TipTiltSource: 'PWFS1',
+  m2TipTiltSource: 'OIWFS',
   m2FocusEnable: true,
-  m2FocusSource: 'PWFS2',
+  m2FocusSource: 'OIWFS',
   m2TipTiltFocusLink: true,
   m2ComaEnable: true,
   m1CorrectionsEnable: true,
   m2ComaM1CorrectionsSource: 'OIWFS',
   mountOffload: true,
   daytimeMode: true,
-  probeTracking: 'PWFS1',
+  probeTracking: 'OIWFS',
```

Comment:
This does not match the options in the UI:
<img width="207" alt="afbeelding" src="https://github.com/user-attachments/assets/8f752a5f-c59c-4e34-a263-e5282ee40674">

`lightPath` is also not used anywhere